server/src/routes/characterRoutes.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express'
+
+import { protect } from '../controllers/authController'
+import { createCharacter } from '../controllers/characterController'
+
+const router = Router()
+
+router.use('/', protect)
+
+router.post('/create-character', createCharacter)
```

Comment:
```suggestion
router.route("/create-character).post(protect, createCharacter)
```

Reply:
So right now i have it set like this as there will be more routes, an update, delete and get, so instead of writing (protect, ...) on each one i wanted to call it once before all the routes. at least thats what Jonas did on his course, do you think it would be better to call protect on each route?

Reply:
Yes that is actually a really good idea, i think for this one line it cuts down on code but if we are planning more here maybe it would be better, for now lets leave it like this